sdk/iot/modelsrepository/src/exceptions.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft.
+// Licensed under the MIT license.
+
+/**
+ * A ModelError will be thrown in the even the Model in the repo is malformed in some standard way.
+ */
+export class ModelError extends Error {
```

Comment:
Doesn't have to be for this beta release, but it would be helpful to have a sample showing how someone might react to seeing a `ModelError` since we're saying it's important enough to distinguish from a regular error.

Reply:
this is fair. I'll talk to the other language people on what kind of sample would be good for this.